include/cookie.php

```diff
@@ -31,7 +32,7 @@ protected static function parse_args( $args ) {
 		$expiration = apply_filters( 'pll_cookie_expiration', YEAR_IN_SECONDS );
```

Comment:
Since we're doing a strict comparison now instead of `>`, it would be good to typecast the filter's output:

```suggestion
		$expiration = (int) apply_filters( 'pll_cookie_expiration', YEAR_IN_SECONDS );
```